packages/react-core/src/components/Card/CardHeader.tsx

```diff
@@ -2,14 +2,27 @@ import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/Card/card';
 import { CardContext } from './Card';
+import { CardHeaderMain } from './CardHeaderMain';
+import { CardActions } from './CardActions';
 import { Button } from '../Button';
 import AngleRightIcon from '@patternfly/react-icons/dist/esm/icons/angle-right-icon';
 
+export interface CardHeaderActionsObject {
+  /** Actions of the card header */
+  actions: React.ReactNode;
+  /** Flag indicating that the actions have no offset */
+  hasNoOffset?: boolean;
+  /** Additional classes added to the action */
```

Comment:
just a nit but this comment is a bit confusing, the classes are added to the `.pf-c-card__actions` element - not any particular action. https://github.com/patternfly/patternfly-react/blob/5f34c07083fd6c707af4f031bb667c8bb6448287/packages/react-core/src/components/Card/CardHeader.tsx#L72

I'm using "wrapper" below, but it could be element, container, or something like that.

```suggestion
  /** Additional classes added to the actions wrapper */
```